Architecture.md

```diff
@@ -182,10 +189,65 @@ As described previously, the API manages most of the user interactions and redir
 
 ### Database architecture
 
-**_TODO: [CLIENT] describe the database architecture, as referenced in the MCD in the README_**
+As the database is unified and serves as a focal point of the application, everything is contained in the same logical database.
```

Comment:
I don't exactly understand the meaning here. Why the database is logical? 
```suggestion
As the database is unified and serves as a focal point of the application, it contains all of the data necessary for the functioning of PaaSTech's different components.
```

Reply:
If you prefer I can remove the logical, however I would rephrase it more like this : 
```
As the database is unified and serves as a focal point of the application, all the data is contained in the same database.
```
Personally I think it is pretty clear that it is all the data for PaaSTech so imo no need to explain that

Reply:
The database is logical, as opposed to physical. The physical database is the instance of the RDBMS, whereas the logical database is a database in that instance